foodsearch/src/components/India/NonVegetarian.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from "react";
+import "./Vegetarian.css";
+import { Link } from "react-router-dom";
+export default function NonVegetarian() {
+  const [advice, setAdvice] = useState([]);
+
+  useEffect(() => {
+    fetch("http://localhost:2000/home?foodtype=NonVegetarian")
```

Comment:
Add `http://localhost:2000` to `.env` this way you don't have to update later when you deploy.